cloud-interfaces/api/cloudnetworks-api.rst

```diff
@@ -145,27 +145,24 @@ In the sample response,
 correspond to the information available on the Cloud Control Panel.
 
 In the
-:rax-docs:`Getting Started Guide for the Cloud Networks API <networks/api/v2/cn-gettingstarted>`,
-you can see an example of
-:rax-docs:`obtaining a list of networks by using the cURL command-line interface (CLI)
-<networks/api/v2/cn-gettingstarted/content/neutron_list_networks_curl.html>`.
+:rax-docs:`Getting Started Guide for the Cloud Networks API <cloud-networks/v2/getting-started/>`,
+you can see an example with the cURL command-line interface (CLI) for
+:rax-docs:`Listing networks (cURL)
+<cloud-networks/v2/getting-started/managing-networks/creating-network-curl/#listing-networks-curl>`.
 
 Learn about Cloud Networks in SDK QuickStart
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 In the
 :rax-dev-quickstart:`SDK QuickStart for Cloud Networks <cloud-networks/getting-started>`,
```

Comment:
This link gives a 404. (And there is the same link below in the paragraph starting with "To see examples" that also gives a 404